### DISCUSSION TITLE
Project 1 Number 3 question

### DISCUSSION THREAD

**Discussion by ecieszynski** [cooperrc/computational-mechanics#34] [Q&A]
I've been trying to debug this for most of the late afternoon and evening, but my plot looks a little too perfect. Definitely made an error somewhere, and just reaching out to ask for some guidance. This question of the homework has been pretty confusing for me.

![image](https://user-images.githubusercontent.com/106542680/172281170-02e7997d-69f8-4f31-b944-97bfa5fb21e7.png)

**Comment by cooperrc** [comment]
Try reducing the number of time steps (equivalently increase time step size). The numerical result should diverge from your analytical result. 

I think the problem is that you're using the analytical answer to solve for the numerical result. 

The numerical solution should use the differential equation,


$T_{i+1}=T_i -kT_i$
